tests/TestCase/I18n/Middleware/LocaleSelectorMiddlewareTest.php

```diff
@@ -101,6 +101,20 @@ public function testInvokeLocaleAccepted()
         $this->assertSame('es', I18n::getLocale(), 'es is accepted');
     }
 
+    /**
+     * The default locale should change when the request locale has an accepted fallback option
+     *
+     * @return void
+     */
+    public function testInvokeLocaleAcceptedFallback()
+    {
+        $request = ServerRequestFactory::fromGlobals(['HTTP_ACCEPT_LANGUAGE' => 'es-ES;q=0.8,da;q=0.4']);
+        $response = new Response();
+        $middleware = new LocaleSelectorMiddleware(['en_CA', 'es']);
```

Comment:
What about the case where `en_US` is supported but the request is for `en_GB`?

Reply:
Per one of the comments on the Locale::lookup man page, this is not a situation that PHP supports. My interpretation is that US English would be an *alternative to* British, not a *fallback for* it. Adding support above and beyond what PHP provides here is certainly possible, but gets more complicated. See http://www.thefutureoftheweb.com/blog/use-accept-language-header for an example that would bypass any usage of PHP's built-in functions for this.

Reply:
Sorry, I missed the earlier reply and didn't think to check on this again.